lib/shaffle.js

```diff
@@ -0,0 +1,14 @@
+export function shaffle(array) {
```

Comment:
Musisz mi to wyjasnic bo nie rozumiem tej funkcji

Reply:
Nie rozumiesz czegos w kodzie czy metody jej działania?

Reply:
slabe nazewnictwo